frontend/package.json

```diff
@@ -51,7 +51,7 @@
     "cypress:delete:reports": "rm cypress/results/* || true",
     "cypress:combine:reports": "jrm cypress/results/combined-report.xml \"cypress/results/*.xml\"",
     "lint": "eslint --ext js,ts,tsx src",
-    "lint:precommit": "eslint -c .eslintrc_precommit $(git diff --staged --name-only HEAD | grep -E '\\.tsx?$' | cut -d'/' -f2-)",
+    "lint:precommit": "eslint -c .eslintrc_precommit $(git diff --staged --name-only HEAD --diff-filter=AM | grep -E '\\.tsx?$' | cut -d'/' -f2-)",
```

Comment:
Filter added to apply precommit lint only to added and modified files (it was throwing lint error because of deleted file)

Reply:
I like these author-written review comments, it's helpful without having to add comments in the code.